src/main/java/net/neoforged/nfrtgradle/NeoFormRuntimePlugin.java

```diff
@@ -0,0 +1,52 @@
+package net.neoforged.nfrtgradle;
```

Comment:
This should be split into `dsl`, `internal`, and `tasks` subpackages similarly to MDG.